links-sqlite3.opam

```diff
@@ -1,21 +1,20 @@
-opam-version: "1.2"
+opam-version: "2.0"
 maintainer: "Jan Stolarek <jan.stolarek@ed.ac.uk>"
 authors: "The Links Team <links-dev@inf.ed.ac.uk>"
+synopsis: "SQLite database driver for the Links Programming Language"
 homepage: "https://github.com/links-lang/links"
 dev-repo: "https://github.com/links-lang/links.git"
 bug-reports: "https://github.com/links-lang/links/issues"
 license: "GPL-2"
 
-available: [
-  ocaml-version >= "4.06.0"
-]
-
 build: [
-  [ "jbuilder" "build" "-p" name "-j" jobs ]
+  [ "dune" "subst" ] {pinned}
+  [ "dune" "build" "-p" name "-j" jobs ]
 ]
 
 depends: [
-  "jbuilder" {build}
+  "ocaml" {>= "4.06.0"}
+  "dune" {build}
   "sqlite3"
```

Comment:
Would be nice to be able to check that the actual installed SQLite version is > 3.25.0, but not sure if OPAM gives us a way to say this.  If not, then row_number queries will just mysteriously fail.

Reply:
Ideally this would tie in with the runtime database driver test above. We don't need 3.25.0 for building, just for shredding.

Reply:
There is a virtual OPAM package `conf-sqlite3`, but it doesn't seem to track the version number of system package.